src/prefect/collection_blocks_data.json

```diff
@@ -0,0 +1,237 @@
+{
```

Comment:
Is this generated through `SnowflakeCredentials.schema()`?

What happens if a new version of the block gets released with new / deleted fields?

Reply:
This is the output of `._to_block_type()` and `.to_block_schema()` with a couple of keys removed to match the shape of `BlockTypeCreate` and `BlockSchemaCreate`. When changes happen to collection blocks, we'll need to generate the necessary data and submit a PR to this file. We'll work to automate as much of those steps as possible.